app/views/publishers/statements/index.html.slim

```diff
@@ -9,31 +9,8 @@ noscript
   )
   = render partial: "choose_channel_button"
 
-/ .single-panel--wrapper.single-panel--wrapper--xlarge#statements_panel
-/   = render "panel_flash_messages"
-/   .single-panel--content
-/     .single-panel--padded-content.text-left
-/       h3= t ".title"
-/       p= t ".description"
-/       .statements-table
-/         .header
-/           .period = t ".period"
-/           .download= t ".download"
-/         - if @statement_has_content
-/           .content#generated_statements
-/             .statement
-/               .period = t(".this_month")
-/               .download = link_to(t(".download"), statement_path(current_publisher, statement_period: "this_month", :"data-piwik-action" => "DownloadThisMonthStatement", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "StatementFlow"), format: :pdf)
-/             .statement
-/               .period = t(".last_month")
-/               .download = link_to(t(".download"), statement_path(current_publisher, statement_period: "last_month", :"data-piwik-action" => "DownloadLastMonthStatement", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "StatementFlow"), format: :pdf)
-/             .statement
-/               .period = t(".all")
-/               .download = link_to(t(".download"), statement_path(current_publisher, statement_period: "all", :"data-piwik-action" => "DownloadAllStatement", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "StatementFlow"), format: :pdf)
-/         - else
-/           .content.empty
 .container.p-4.bg-white.shadow.rounded
-  - unless @uphold_connection.can_read_transactions?
+  - if @cannot_read_uphold_transactions
```

Comment:
Actually this part seems to be undefined. If you want to move the logic from the controller to the view, that's fine by me